server/controllers/wazuh-reporting.ts

```diff
@@ -201,57 +191,44 @@ export class WazuhReportingCtrl {
    * @param {Array<Strings>} ids ids of agents
    * @param {String} apiId API id
    */
-  private async buildAgentsTable(context, printer: ReportPrinter, agentIDs: string[], apiId: string, multi = false) {
+  private async buildAgentsTable(context, printer: ReportPrinter, agentIDs: string[], apiId: string, groupID: string | null = null) {
```

Comment:
Be careful using `null`

That `null = null` does not look very well.

Reply:
Code:
```
groupID: string | null = null
```

- Parameter definition schema
```
parameter name : TS type definition = default value
```

This means:
- parameter name: `groupID`
- type definition is: `string | null`
- default value: `null`

Reply:
And wouldn't it be better to simply have a single type, which is string, and set its default value to an empty string?

Using combined types for a simple parameter as that one seems overcomplicated.

Reply:
Yes, it is another option. I will change it.

Reply:
Changed. Thank you for the suggestion!